tests/platform_tests/test_engine.py

```diff
@@ -20,7 +20,11 @@
 import random
 import time
 
-from tank_test.tank_test_base import TankTestBase, skip_if_pyside_missing
+from tank_test.tank_test_base import (
```

Comment:
Is that black formatting it like that?
I've not seen it formatted that way before.